pkg/nuclide/file-tree/spec/FileTreeController-spec.js

```diff
@@ -66,6 +66,21 @@ describe('FileTreeController', () => {
     fileTreeController.destroy();
   });
 
+  it('checks if core:backspace and core:delete is registered on .nuclide-file-tree', () => {
+    var timesCalled = 0;
+    // Find div element
+    var el = fileTreeController._panelController._hostEl.getElementsByClassName('nuclide-file-tree')[0];
+    // mock deleteSelection
+    fileTreeController.deleteSelection = () => {
```

Comment:
Jasmine provides a concept for this called [spies](https://jasmine.github.io/1.3/introduction.html#section-Spies) that does roughly what you did but ensures the methods are restored after each spec. It would look something like this:

```
spyOn(fileTreeController, 'deleteSelection');

atom.commands.dispatch(el, 'core:backspace');
atom.commands.dispatch(el, 'core:delete');

expect(fileTreeController.deleteSelection.calls.length).toEqual(2);
```